ha/monitor/k8s/monitor.py

```diff
@@ -15,35 +15,104 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
+"""
+   Monitoring and processing the Kubernetes events
+"""
 
-from ha.monitor.k8s.object_monitor import ObjectMonitor
+import os
+import signal
+from cortx.utils.log import Log
 
 from cortx.utils.conf_store import Conf
+from ha.k8s_setup.const import _DELIM
 from ha import const
 from ha.core.config.config_manager import ConfigManager
+from ha.util.message_bus import MessageBus
+from ha.monitor.k8s.object_monitor import ObjectMonitor
+from ha.monitor.k8s.const import K8SClientConst
+
+class ResourceMonitor:
+    """
+    Manages the Monitor threads
+    """
+    def __init__(self, wait_time=10):
+        """
+        Init method
+        Create monitor objects and Sets the callbacks to sigterm
+        """
+        # set sigterm handler
+        signal.signal(signal.SIGTERM, self.set_sigterm)
+
+        # Read I/O pod selector label from ha.conf . Will be received from provisioner confstore
+        # provisioner needs to be informed to add it in confstore  (to be added there )
+        ConfigManager.init("k8s_resource_monitor")
+
+        # event output in pretty format
+        kwargs = {K8SClientConst.PRETTY : True}
+
+        # Seting a timeout value, 'timout_seconds', for the stream.
+        # timeout value for connection to the server
+        # If do not set then we will not able to stop immediately,
+        # becuase synchronus function watch.stream() will not come back
+        # until catch any event on which it is waiting.
+        kwargs[K8SClientConst.TIMEOUT_SECONDS] = K8SClientConst.VAL_WATCH_TIMEOUT_DEFAULT
+
+        # Get MessageBus producer object for all monitor threads
+        producer = self._get_producer()
+
+        # Change to multiprocessing
+        # Creating NODE monitor object
+        self.node_monitor = ObjectMonitor(producer, K8SClientConst.NODE, **kwargs)
+
+        pod_labels = Conf.get(const.HA_GLOBAL_INDEX, "data_pod_label")
+        pod_label_str = ', '.join(pod_label for pod_label in pod_labels)
+        # TODO : Change 'name' field to 'app' in label_selector if required.
+        kwargs[K8SClientConst.LABEL_SELECTOR] = f'name in ({pod_label_str})'
+
+        # Creating POD monitor object
+        self.pod_monitor = ObjectMonitor(producer, K8SClientConst.POD, **kwargs)
+
+    def _get_producer(self):
+        """
+        Get message bus producer
+        """
+        message_type = Conf.get(const.HA_GLOBAL_INDEX, f"MONITOR{_DELIM}message_type")
+        producer_id = Conf.get(const.HA_GLOBAL_INDEX, f"MONITOR{_DELIM}producer_id")
+        Log.info("Initializing message bus.")
```

Comment:
The Guidance from Ujjwal is that there should be no logs in the system which do not print any variable data. 
Please remove this log, the log on line 83 will suffice; you can append this message to that log if needed. 

Please remove/modify any similar logs from other places also